transmart-api-server/build.gradle

```diff
@@ -28,8 +28,9 @@ dependencies {
     compile project(':transmart-rest-api')
     compile project(':transmart-notifications')
 
-    compile "org.springframework.boot:spring-boot-starter-security"
```

Comment:
By removing this dependency we started to rely on transitive dependency in the code. Not desired scenario. I am going to fix this.